concent_api/core/subtask_helpers.py

```diff
@@ -131,7 +132,13 @@ def update_timed_out_subtask(subtask: Subtask) -> None:
     )
 
 
-def update_timed_out_subtasks_in_message(client_message: Message, client_public_key: bytes) -> None:
+def update_subtasks_from_incoming_message_if_timed_out(client_message: Message, client_public_key: bytes) -> None:
```

Comment:
Also here. This change is not related with changes which you are doing in this PR

Reply:
as above